Resources/Prototypes/Recipes/Construction/utilities.yml

```diff
@@ -36,11 +37,29 @@
   graph: cable_terminal
   startNode: start
   targetNode: cable_terminal
-  category: Utility
+  category: Utilities
   description: "Input of devices such as the SMES. The red cables needs to face the device."
   icon:
     sprite: Structures/Power/cable_terminal.rsi
     state: term
   objectType: Structure
   placementMode: SnapgridCenter
   canBuildInImpassable: false
+
+
+#DISPOSALS
+- type: construction
+  name: disposal unit
+  id: DisposalUnit
+  graph: disposalMachine
+  startNode: start
+  targetNode: disposal_unit
+  category: Utilities
+  placementMode: SnapgridCenter
+  canBuildInImpassable: false
+  icon:
+    sprite: Structures/Piping/disposal.rsi
+    state: "disposal"
+
+
+#ATMOS
```

Comment:
ATMOS?

Reply:
For future graphs of atmos devices like air alarm, fire alarm etc. Thought since I axed disposals having its own category then atmos could come under here too.